test/Microsoft.AspNet.Mvc.ViewFeatures.Test/ViewComponentResultTest.cs

```diff
@@ -572,30 +573,30 @@ public IEnumerable<ViewComponentDescriptor> GetViewComponents()
             }
         }
 
-        private class TextViewComponent : ViewComponent
+        private static string ReadBody(HttpResponse response)
         {
-            public HtmlString Invoke(string name)
+            response.Body.Seek(0, SeekOrigin.Begin);
+
+            using (var reader = new StreamReader(response.Body))
             {
-                return new HtmlString("Hello, " + name);
+                return reader.ReadToEnd();
             }
         }
+    }
 
-        private class AsyncTextViewComponent : ViewComponent
+    public class TextViewComponent : ViewComponent
```

Comment:
needed to change this as part of the check that I introduced on DefaultViewComponentActivator, I think this makes the test more compliant with the expectations we have for view components